admin_panel/src/commonMain/kotlin/net/thechance/mena/admin_panel/presentation/component/TableHeaderRow.kt

```diff
@@ -51,47 +51,41 @@ fun TableHeaderRow(
             color = Theme.colorScheme.shadePrimary,
             modifier = Modifier.widthIn(min = 78.dp)
         )
-
         SortableHeaderCell(
             text = stringResource(Res.string.user_name),
             sortType = UsersManagementScreenState.SortType.USERNAME,
             currentSort = sortState,
             onSortClicked = onSortClicked,
-            modifier = Modifier.widthIn(min = 268.dp)
+            modifier = Modifier.widthIn(min = 268.dp, max = 268.dp)
```

Comment:
If the min and max is 268 we can just give it a fixed width 
```suggestion
            modifier = Modifier.width(268.dp)
```